### PR TITLE
Update feeds Reliability - 2

### PR DESCRIPTION
**Pull request by GuyAfik**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
[link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-3646)

## Description
add Reliability

### CODE REVIEW

File: Packs/CTIX/Integrations/CTIX/CTIX.yml [hunk 1]
```diff
@@ -31,6 +31,25 @@ configuration:
...
+  - A - Completely reliable
+  - B - Usually reliable
+  - C - Fairly reliable
+  - D - Not usually reliable
+  - E - Unreliable
+  - F - Reliability cannot be judged
+  required: false
+  type: 15
+- display: ""
+  name: feedExpirationPolicy
```
Comment (JudahSchwartz) [inline Packs/CTIX/Integrations/CTIX/CTIX.yml:49]: arent we missing this here? 
options:
  - never
  - interval
  - indicatorType
  - suddenDeath

**Review by JudahSchwartz — APPROVED** [review]
please add feedExpirationPolicy options.
Looks good otherwise

File: Packs/CTIX/Integrations/CTIX/CTIX.yml (hunk 1 above)
Comment (GuyAfik) [reply Packs/CTIX/Integrations/CTIX/CTIX.yml:49]: checking with Dror

Comment (GuyAfik) [reply Packs/CTIX/Integrations/CTIX/CTIX.yml:49]: he said the way i did is good enough, @Ni-Knight FYI

**Comment by GuyAfik** [comment]
> please add feedExpirationPolicy options. Looks good otherwise

fyi - https://github.com/demisto/content/pull/20350#discussion_r936510791

File: Packs/CTIX/Integrations/CTIX/CTIX.yml (hunk 1 above)
Comment (Ni-Knight) [reply Packs/CTIX/Integrations/CTIX/CTIX.yml:49]: It should be enough, since up to today it had none.

**Comment by GuyAfik** [comment]
> please add feedExpirationPolicy options. Looks good otherwise

done

**Pull request merged**